src/components/Select.js

```diff
@@ -0,0 +1,98 @@
+import React from 'react'
+import styled from 'styled-components'
+
+const SelectWrapper = styled.span`
+  display: inline-block;
+  border: 2px solid ${p => p.theme.colors.default};
+  border-radius: 8px;
+  padding: 8px 24px 8px 16px;
+  margin: 8px;
+  ${p =>
+    p.disabled
+      ? `
+  opacity: ${p.theme.opacity.disabled};
+  cursor: not-allowed;
+`
+      : `:hover {
+  background: ${p.theme.colors.selects.hover};
+  border: 2px solid ${p.theme.colors.primary};
+  cursor: pointer;
+}`}
+  ${p =>
+    p.checked &&
+    `
+  background: ${p.theme.colors.primary};
+  border: 2px solid ${p.theme.colors.primary};
+  transition: background-color 0.25s linear;
+  :hover {
+    background: ${p.theme.colors.selects.focus};
+  `}
+  ${p => `:focus-within {
+    background: ${p.theme.colors.selects.focus};
+    border: 2px solid ${p.theme.colors.primary};
+  }
+  :focus {
+    background: ${p.theme.colors.selects.focus};
+    border: 2px solid ${p.theme.colors.primary};
+  }`}
```

Comment:
`:focus` and `:focus-within` are a little repetitive - anyone have suggestions on how to combine these lol

Reply:
you could do something like CardLike in Common.js where you set the css to a variable (could be a variable in this file, or in Common.js if you foresee us reusing the styles), then use it like how ${CardLike} is called :)